packages/vaadin-grid/src/vaadin-grid-column.js

```diff
@@ -67,7 +67,8 @@ export const ColumnBaseMixin = (superClass) =>
          * When set to true, the cells for this column are hidden.
          */
         hidden: {
-          type: Boolean
+          type: Boolean,
+          value: false
```

Comment:
This property is also shadowing a native one. I used `false` instead of undefined as default and adapted two failing tests. Not 100% sure about the implications.